tests/components/vizio/conftest.py

```diff
@@ -53,7 +53,7 @@ def vizio_get_unique_id_fixture():
     """Mock get vizio unique ID."""
     with patch(
         "homeassistant.components.vizio.config_flow.VizioAsync.get_unique_id",
-        return_value=UNIQUE_ID,
+        AsyncMock(return_value=UNIQUE_ID),
```

Comment:
`patch` will in Python 3.8+ automatically patch a target coroutine function with an `AsyncMock`. Why doesn't that work on Travis?

Reply:
No idea. Travis logs say tests are run with 3.8.1 in case that makes a difference. Haven't tried with any 3.8.x in local tox.

Reply:
Tests pass here locally without this patch. What's up with Travis?

Reply:
The target method is a static method. That patching was fixed in Python 3.8.2:
https://bugs.python.org/issue39082

Reply:
Adjusted commit message accordingly.

Reply:
I'd suggest updating the Travis Python version to at least Python 3.8.2 instead, if possible.

https://docs.travis-ci.com/user/languages/python/#python-versions

Reply:
If 3.8.1 is a version HA is supposed to work with, I don't think it makes sense to jump to some other version and pretend the problem doesn't exist. But if it's not, it should be clearly marked as such, and _then_ adjust envs accordingly.

Anyway I think the fix here is so simple that we shouldn't be wasting this much time on it :)

Reply:
Agreed. Our minimum version is Python 3.8.0 and so we should actually adjust our testing env to test with that.